Makefile

```diff
@@ -1529,12 +1529,18 @@ else
 	ARCH := $(shell getconf LONG_BIT)
 endif
 
+ifneq ($(wildcard /lib/libc.musl-x86_64.so.1),)
```

Comment:
@adamretter I need to change this condition to something like `/lib/libc.musl*.so*`. Will that work for ppc64le?

Reply:
Hmmm I am a little worried about the fragility there also, what if musl is installed somewhere else? Is there a better way to detect it?

Reply:
I looked at a node.js library that detects libc variants and there are two checks before the filesystem check:
1. `getconf GNU_LIBC_VERSION` (I could not find a similar MUSL_LIBC_VERSION variable)
2. `ldd /bin/sh | grep 'musl'`

I prefer the `ldd` check. The first one assumes if it's not glibc, assume musl.